Dockerfile

```
@@ -1,12 +1,12 @@
-FROM node
+FROM node:12.18.1
```

I would suggest to skip the minor and patch versions since the major would be sufficient in most of cases.
```suggestion
FROM node:12
```

---

Done